contracts/token/ERC721/extensions/ERC721Pausable.sol

```diff
@@ -27,14 +27,12 @@ abstract contract ERC721Pausable is ERC721, Pausable {
      *
      * - the contract must not be paused.
      */
-    function _beforeTokenTransfer(
+    function _update(
         address from,
         address to,
-        uint256 firstTokenId,
-        uint256 batchSize
+        uint256 tokenId
     ) internal virtual override {
-        super._beforeTokenTransfer(from, to, firstTokenId, batchSize);
-
         _requireNotPaused();
```

Comment:
I think it's a good moment to use `whenNotPaused` instead as suggested [here](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4261#discussion_r1212762822)

Reply:
Personally I prefer the function call. They are functionally equivalent but the function call is visible inside the function, I just find it more explicit. I don't know if everyone feels the same though.

Reply:
I prefer `whenNotPaused` precisely because it's less explicit (== less pervasive?).

I'm fine with the function call if we agree to keep consistency between this and [ERC1155Pausable](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/bb64458928d08759e1f36cc0c11a8e76c976de0c/contracts/token/ERC1155/extensions/ERC1155Pausable.sol) where we use the modifier instead.